client/src/components/Image.jsx

```diff
@@ -1,24 +1,61 @@
 import React from 'react';
 import styled from 'styled-components';
+import Popup from 'reactjs-popup';
+import { FaSistrix } from 'react-icons/fa';
```

Comment:
maybe i should use this library too!

Reply:
@masoncpott 'react-icons' is really cool. Also, the best part is that you can pic and choose which one to import, vs bringing over the entire library. So it helps with your app performance.